wpilibjExamples/src/main/java/edu/wpi/first/wpilibj/examples/rapidreactcommandbot/subsystems/Drive.java

```diff
@@ -98,4 +128,26 @@ public Command driveDistanceCommand(double distanceMeters, double speed) {
         // Stop the drive when the command ends
         .finallyDo(interrupted -> m_drive.stopMotor());
   }
+
+  /**
+   * Returns a command that turns to robot to the specified angle using a motion profile and PID
+   * controller.
+   *
+   * @param angleDeg The angle to turn to
+   */
+  public Command turnToAngleCommand(double angleDeg) {
+    return startRun(
+            () -> m_controller.reset(m_gyro.getRotation2d().getDegrees()),
+            () ->
+                m_drive.arcadeDrive(
+                    0,
+                    m_controller.calculate(m_gyro.getRotation2d().getDegrees(), angleDeg)
+                        // Divide feedforward voltage by battery voltage to normalize it to [-1, 1]
+                        + m_feedforward
+                                .calculate(RadiansPerSecond.of(m_controller.getSetpoint().velocity))
+                                .in(Volts)
```

Comment:
The controller is using degrees and degrees per second based on the constant names. 

I also don't think the unit library needs to be used here, but I suppose we have to since the double functions are deprecated out of necessity.